packages/Calendar/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@tds/community-calendar",
+  "version": "1.0.0-0",
+  "description": "",
+  "main": "index.cjs.js",
+  "module": "index.es.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/telus/tds-community.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "author": "TELUS digital",
+  "engines": {
+    "node": ">=8"
+  },
+  "bugs": {
+    "url": "https://github.com/telus/tds-community/issues"
+  },
+  "homepage": "http://tds.telus.com",
+  "peerDependencies": {
+    "react": "^16.8.2",
+    "react-dom": "^16.8.2",
+    "styled-components": "^4.1.3"
+  },
+  "dependencies": {
+    "@tds/core-colours": "^2.1.0",
+    "@tds/core-decorative-icon": "^1.2.0",
+    "@tds/core-responsive": "^1.1.0",
+    "@tds/shared-typography": "^1.3.3",
+    "moment": "^2.24.0",
+    "prop-types": "^15.6.2",
+    "react-dates": "^19.0.3",
+    "react-moment-proptypes": "^1.6.0"
+  },
+  "devDependencies": {
+    "@tds/shared-safe-rest": "^1.0.0"
```

Comment:
Please use `safeRest` from `@tds/util-helpers`. See https://github.com/telus/tds-core/blob/master/packages/Button/Button.jsx#L11 for an example.